build.gradle

```diff
@@ -20,7 +20,7 @@ publishing {
 }
 
 group = 'com.github.hmcts'
-version = '2.1.18'
+version = '2.1.19'
```

Comment:
This will now need updating to 20 following recent merge

Reply:
Have updated to 2.1.20 and all backend services using it